<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>EPROM Programmer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="serialPort">
         <property name="title">
          <string>Serial port</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="3">
           <widget class="QListWidget" name="portList">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="connectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="disconnectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="updateButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Update list</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="chipType">
         <property name="title">
          <string>Chip type</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="c16Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C16</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="c32Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C32</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="c128Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C128</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QRadioButton" name="c256Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C256</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="c64Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C64</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QRadioButton" name="c801Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C801</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="c4001Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C4001</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="c2001Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C2001</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="c1001Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C1001</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="c512Button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>27C512</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="8">
           <widget class="QPushButton" name="readChipButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Read chip</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="saveFileButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8">
           <widget class="QPushButton" name="writeChipButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Write chip</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="openFileButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Open file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="verifyChipButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Verify chip</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="showButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show buffer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="voltageChipButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Voltage</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
